pkg/verify/verify.go

```diff
@@ -30,6 +30,17 @@ type Interface interface {
 	Verify(ctx context.Context, releaseDigest string) error
 }
 
+// SignatureStore retrieves signatures for a digest or returns an error. It requires String()
+// for describing the store.
+type SignatureStore interface {
+	// DigestSignatures returns zero or more signatures for the provided digest, or returns an
+	// error if no signatures could be retrieved.
```

Comment:
maybe obvious, but probably still worth calling out that the store is not required to perform any internal signature validation.